packages/astro/src/assets/services/squoosh.ts

```diff
@@ -95,12 +95,22 @@ const service: LocalImageService = {
 			}
 		}
 
-		const data = await processBuffer(inputBuffer, operations, format, quality);
+		try {
+			const data = await processBuffer(inputBuffer, operations, format, quality);
 
-		return {
-			data: Buffer.from(data),
-			format: format,
-		};
+			return {
+				data: Buffer.from(data),
+				format: format,
+			};
+		} catch (err) {
+			const message = err && typeof err === "object" && "message" in err ? err.message : err;
+			// Edge-case for unsupported formats, as Squoosh's error message is not very helpful
+			if (message === "Buffer has an unsupported format") {
```

Comment:
This check is very fragile, because if upstream decides to change the error message , even inside a `patch`, we won't be able to show a proper message anymore. But I suppose we can't do much here, right?

Reply:
Squoosh is deprecated, unmaintained and vendored by us. So there won't be an upstream change anyway